Makefile

```diff
@@ -495,4 +496,26 @@ validate-release: setup-venv
 ## prepare files for OperatorHub PR
 ## use this target when the operator needs to be released as upstream operator
 prepare-operatorhub-pr:
-	./hack/prepare-operatorhub-pr.sh $(OPERATOR_VERSION) $(OPERATOR_BUNDLE_IMAGE_REF)
\ No newline at end of file
+	./hack/prepare-operatorhub-pr.sh $(OPERATOR_VERSION) $(OPERATOR_BUNDLE_IMAGE_REF)
+
+.PHONY: prepare-release-manifests
+## prepare file for release
+## use this target when the operator needs to be released as a single manifest with project release tags
+prepare-release-manifests: setup-venv
+	./hack/prepare-operatorhub-pr.sh $(OPERATOR_VERSION) $(OPERATOR_BUNDLE_IMAGE_REF)
+	$(Q)$(PYTHON_VENV_DIR)/bin/pip install yq==2.10.0
+	$(eval CONTAINER_IMAGE := $(shell $(PYTHON_VENV_DIR)/bin/yq -r  .metadata.annotations.containerImage ./out/operatorhub-pr-files/service-binding-operator/$(OPERATOR_VERSION)/manifests/$(OPERATOR_BUNDLE_IMAGE_REF).clusterserviceversion.yaml))
+	$(Q)echo $(CONTAINER_IMAGE)
+	mkdir -p ./out/releases/$(OPERATOR_VERSION)
+	touch ./out/releases/$(OPERATOR_VERSION)/release.yaml
+	cp ./out/operatorhub-pr-files/service-binding-operator/$(OPERATOR_VERSION)/manifests/operators.coreos.com_servicebindings_crd.yaml $(RELEASE_MANIFEST)/release.yaml 
+	printf '\n---\n' >> $(RELEASE_MANIFEST)/release.yaml
+	cat ./deploy/operator.yaml >> $(RELEASE_MANIFEST)/release.yaml
+	printf '\n---\n' >> $(RELEASE_MANIFEST)/release.yaml
+	sed -i -e 's,REPLACE_IMAGE,$(CONTAINER_IMAGE),g' $(RELEASE_MANIFEST)/release.yaml
+	cat ./deploy/role_binding.yaml >> $(RELEASE_MANIFEST)/release.yaml
+	printf '\n---\n' >> $(RELEASE_MANIFEST)/release.yaml
+	cat ./deploy/role.yaml >> $(RELEASE_MANIFEST)/release.yaml
+	printf '\n---\n' >> $(RELEASE_MANIFEST)/release.yaml
+	cat ./deploy/service_account.yaml >> $(RELEASE_MANIFEST)/release.yaml
+	printf '\n---\n' >> $(RELEASE_MANIFEST)/release.yaml
```

Comment:
* Please move it into a separate script, for easier maintainability. 
* Consider using Python for that task
* Extract from CSV file full resources, try to copy from `deploy` folder only things that cannot be found in CSV (e.g. service accounts)

Reply:
role bindings as well.